src/routes/ordersRouter.ts

```diff
@@ -0,0 +1,75 @@
+import express, { NextFunction, Request, Response } from "express";
+import { IFullOrder, IOrder, IProduct } from "../db/modelsType";
+import { orderService, productService } from "../services";
+
+const ordersRouter = express.Router();
+
+ordersRouter.get(
+  "/:id",
+  async function (req: Request, res: Response, next: NextFunction) {
+    const userId = Number(req.params.id);
+    try {
+      const ordersList: IOrder[] = await orderService.getOrdersByUserId(userId);
```

Comment:
`orders` or `orderList`, better `orders`.
```suggestion
      const orders: IOrder[] = await orderService.getOrdersByUserId(userId);
```